default/generated/spring-framework/spring-framework-5.x-to-6.0-data-access.yaml

```diff
@@ -90,3 +90,39 @@
       url: https://eclipse.dev/eclipselink/downloads/previous_releases.php
     - title: 'Spring 6.0 migration guide'
       url: https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-6.0-Release-Notes#removed-apis
+
+- ruleID: spring-framework-5.x-to-6.0-data-access-00030
+  category: mandatory
```

Comment:
This would be an optional issue instead of a mandatory one, since these members are only deprecated, not removed, so there is no real need to change them yet.